cfgov/regulations3k/views.py

```diff
@@ -1,6 +1,6 @@
+# from django.http import Http404
```

Comment:
Should we remove this commented line of code?

Reply:
Good catch! Removed!